bindata/v4.1.0/kube-apiserver/defaultconfig.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: kubecontrolplane.config.openshift.io/v1
 kind: KubeAPIServerConfig
 admission:
-  pluginConfigs:
+  pluginConfig:
```

Comment:
looking further down:

    network.openshift.io/RestrictedEndpointsAdmission:
      configuration:
        apiVersion: network.openshift.io/v1
        kind: RestrictedEndpointsAdmissionConfig
        restrictedCIDRs:
        - 10.3.0.0/16 # ServiceCIDR
        - 10.2.0.0/16 # ClusterCIDR

neither of those CIDRs is correct. I think the values here will just get overwritten at runtime with the correct values read from the network config status, but it's weird to have explicitly wrong values in there by default...

Reply:
@danwinship what are the correct defaults?

Reply:
we will remove this config and wait for network operator to set it.

Reply:
To be clear, he's talking about this code here: https://github.com/openshift/cluster-kube-apiserver-operator/tree/master/pkg/operator/configobservation/network

Reply:
`172.30.0.0/16` for ServiceCIDR and `10.128.0.0/14` for ClusterCIDR, but they can be overridden from install-config.yaml

Reply:
@mfojtik is that code typo'd too? https://github.com/openshift/cluster-kube-apiserver-operator/blob/master/pkg/operator/configobservation/network/observe_network.go#L18

Reply:
yeah, that looks bugged as well... fixed.